src/lib/services/feature-toggle-service.ts

```diff
@@ -302,6 +306,7 @@ class FeatureToggleService {
                     projectId,
                     featureName,
                     environment,
+                    name: 'Some-Name',
```

Comment:
Same as above. Ideally our spec work wouldn't necessitate adding this field here. We could keep the Schema types out of the stores if they make things tricky, limiting their influence to handlers. 

And could be strategyName if anything.